src/EventService/Controllers/CategoryController.cs

```diff
@@ -19,5 +20,14 @@ public class CategoryController : ControllerBase
   {
     return await command.ExecuteAsync(request);
   }
+  
+  [HttpPost("find")]
```

Comment:
Post?

Reply:
Yeah, Liza has already pointed out this mistake. I will take it into account